src/components/Tokens/TokenDetails/TransactionsTable.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { screen } from '@testing-library/react'
+import { render } from 'test-utils/render'
+
+import { TransactionsTable } from './TransactionsTable'
+
+describe('TransactionsTable', () => {
+  const BITCOIN_ADDRESS = '0x72e4f9f808c49a2a61de9c5896298920dc4eeea9'
+
+  it('shows all columns for extra large breakpoint', () => {
+    window.innerWidth = 1280
+
+    render(<TransactionsTable referenceTokenAddress={BITCOIN_ADDRESS} />)
+
+    expect(screen.getByText('Time')).toBeInTheDocument()
+    expect(screen.getByText('Type')).toBeInTheDocument()
+    expect(screen.getByText('Amount')).toBeInTheDocument()
+    expect(screen.getByText('For')).toBeInTheDocument()
+    expect(screen.getByText('USD')).toBeInTheDocument()
+    expect(screen.getByText('Maker')).toBeInTheDocument()
+  })
+
+  it('shows all columns for large breakpoint', () => {
+    window.innerWidth = 1024
+
+    render(<TransactionsTable referenceTokenAddress={BITCOIN_ADDRESS} />)
+
+    expect(screen.getByText('Time')).toBeInTheDocument()
+    expect(screen.getByText('Type')).toBeInTheDocument()
+    expect(screen.getByText('Amount')).toBeInTheDocument()
+    expect(screen.getByText('For')).toBeInTheDocument()
+    expect(screen.getByText('USD')).toBeInTheDocument()
+  })
+
+  it('hides some columns at medium breakpoint', () => {
+    window.innerWidth = 768
+
+    render(<TransactionsTable referenceTokenAddress={BITCOIN_ADDRESS} />)
+
+    expect(screen.getByText('Time')).toBeInTheDocument()
+    expect(screen.getByText('Type')).toBeInTheDocument()
+    expect(screen.getByText('Amount')).toBeInTheDocument()
+    expect(screen.getByText('USD')).toBeInTheDocument()
+  })
+
+  it('hides some columns at small breakpoint', () => {
+    window.innerWidth = 640
+
+    render(<TransactionsTable referenceTokenAddress={BITCOIN_ADDRESS} />)
+
+    expect(screen.getByText('Time')).toBeInTheDocument()
+    expect(screen.getByText('Type')).toBeInTheDocument()
+    expect(screen.getByText('Amount')).toBeInTheDocument()
+  })
```

Comment:
these tests are identical to Table/index.test.tsx and don't really add value

Reply:
After implementing horizontal scrolling instead of this old scheme of hiding/showing columns, I was able to significantly simplify the tests and remove a lot of duplication.